config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'users#index'
```

Comment:
I think changing this route for testing is fine. But I can't really see why we would want the root of our website to be a list of users (im open to discussing this).

Likely we want to leave root as the default for now, until we create a PagesController to handle static pages like our homepage